docs/01-app/03-api-reference/04-functions/expirePath.mdx

```diff
@@ -0,0 +1,123 @@
+---
+title: expirePath
+description: API Reference for the expirePath function.
+---
+
+`expirePath` allows you to purge [cached data](/docs/app/building-your-application/caching) on-demand for a specific path.
+
+> **Good to know**:
+>
+> - `expirePath` is available in both [Node.js and Edge runtimes](/docs/app/building-your-application/rendering/edge-and-nodejs-runtimes).
+> - `expirePath` only invalidates the cache when the included path is next visited. This means calling `expirePath` with a dynamic route segment will not immediately trigger many revalidations at once. The invalidation only happens when the path is next visited.
+> - Currently, `expirePath` invalidates all the routes in the [client-side Router Cache](/docs/app/building-your-application/caching#client-side-router-cache) when used in a server action. This behavior is temporary and will be updated in the future to apply only to the specific path.
+> - Using `expirePath` invalidates **only the specific path** in the [server-side Route Cache](/docs/app/building-your-application/caching#full-route-cache).
+
+## Parameters
+
+```tsx
+expirePath(path: string, type?: 'page' | 'layout'): void;
+```
+
+- `path`: Either a string representing the filesystem path associated with the data you want to revalidate (for example, `/product/[slug]/page`), or the literal route segment (for example, `/product/123`). Must be less than 1024 characters. This value is case-sensitive.
+- `type`: (optional) `'page'` or `'layout'` string to change the type of path to revalidate. If `path` contains a dynamic segment (for example, `/product/[slug]/page`), this parameter is required. If path refers to the literal route segment, e.g., `/product/1` for a dynamic page (e.g., `/product/[slug]/page`), you should not provide `type`.
+
+## Returns
+
+`expirePath` does not return a value.
+
+## Examples
+
+### Revalidating A Specific URL
+
+```ts
+import { expirePath } from 'next/cache'
+expirePath('/blog/post-1')
+```
+
+This will revalidate one specific URL on the next page visit.
+
+### Revalidating A Page Path
+
+```ts
+import { expirePath } from 'next/cache'
+expirePath('/blog/[slug]', 'page')
+// or with route groups
+expirePath('/(main)/blog/[slug]', 'page')
+```
+
+This will revalidate any URL that matches the provided `page` file on the next page visit. This will _not_ invalidate pages beneath the specific page. For example, `/blog/[slug]` won't invalidate `/blog/[slug]/[author]`.
+
+### Revalidating A Layout Path
+
+```ts
+import { expirePath } from 'next/cache'
+expirePath('/blog/[slug]', 'layout')
+// or with route groups
+expirePath('/(main)/post/[slug]', 'layout')
+```
+
+This will revalidate any URL that matches the provided `layout` file on the next page visit. This will cause pages beneath with the same layout to revalidate on the next visit. For example, in the above case, `/blog/[slug]/[another]` would also revalidate on the next visit.
```

Comment:
```suggestion
This will purge the cache on any URL that matches the provided `layout` file on the next page visit. This will cause pages beneath with the same layout to revalidate on the next visit. For example, in the above case, `/blog/[slug]/[another]` would also revalidate on the next visit.
```